crates/core/src/factories/mem_blocks/test.rs

```diff
@@ -0,0 +1,80 @@
+use kitsune2_api::{BlockTarget, Blocks};
+
+use super::MemBlocks;
+
+#[tokio::test]
+async fn unblocked_target_reports_is_blocked_false() {
+    let blocks = MemBlocks::default();
+    let target =
+        BlockTarget::Agent(bytes::Bytes::from_static(b"test-agent").into());
```

Comment:
Use the test agent id here and in the following tests?
```suggestion
        BlockTarget::Agent(AGENT_1);
```

Reply:
Sweet!